receivers/mqtt/testing.go

```diff
@@ -0,0 +1,16 @@
+package mqtt
+
+// FullValidConfigForTesting is a string representation of a JSON object that contains all fields supported by the notifier Config. It can be used without secrets.
+const FullValidConfigForTesting = `{
+	"brokerUrl": "tcp://localhost:1883",
+	"topic": "grafana/alerts",
+	"messageFormat": "json",
+	"clientId": "grafana-test-client-id",
+	"username": "test-username",
```

Comment:
can you please add password here too (this is full config). I rely upon all fields specified in one of my currently opened PRs